### ISSUE TITLE
Error when running command on my project.

### ISSUE THREAD

**Issue by Webonaute** [symfony/thanks#58]
If a package repository cannot be resolved, the thank command fail and not complete other repositories thank.
```
composer thanks --dry-run -vvv
Reading ./composer.json
Loading config file /Users/dev/.composer/config.json
Loading config file /Users/dev/.composer/auth.json
Loading config file ./composer.json
Checked CA file /private/etc/ssl/cert.pem: valid
Executing command (/Users/dev/projects/myproject): git branch --no-color --no-abbrev -v
Reading /Users/dev/.composer/composer.json
Loading config file /Users/dev/.composer/config.json
Loading config file /Users/dev/.composer/auth.json
Loading config file /Users/dev/.composer/composer.json
Loading config file /Users/dev/.composer/auth.json
Reading /Users/dev/.composer/auth.json
Reading /Users/dev/projects/myproject/vendor/composer/installed.json
Reading /Users/dev/.composer/vendor/composer/installed.json
Loading plugin cweagans\Composer\Patches
Loading plugin PackageVersions\Installer
Loading plugin Symfony\Flex\Flex
Composer >=1.7 not found, downloads will happen in sequence
Loading plugin Symfony\Thanks\Thanks
Running 1.6.5 (2018-05-04 11:44:59) with PHP 7.2.5 on Darwin / 17.5.0
Reading ./composer.json
Downloading https://api.github.com/graphql


  [Composer\Downloader\TransportException]
  Could not resolve to a Repository with the name 'jmespath'.


Exception trace:
 () at /Users/dev/.composer/vendor/symfony/thanks/src/Command/ThanksCommand.php:222
 Symfony\Thanks\Command\ThanksCommand->callGitHub() at /Users/dev/.composer/vendor/symfony/thanks/src/Command/ThanksCommand.php:171
 Symfony\Thanks\Command\ThanksCommand->execute() at phar:///usr/local/Cellar/composer/1.6.4/bin/composer/vendor/symfony/console/Command/Command.php:242
 Symfony\Component\Console\Command\Command->run() at phar:///usr/local/Cellar/composer/1.6.4/bin/composer/vendor/symfony/console/Application.php:843
 Symfony\Component\Console\Application->doRunCommand() at phar:///usr/local/Cellar/composer/1.6.4/bin/composer/vendor/symfony/console/Application.php:193
 Symfony\Component\Console\Application->doRun() at phar:///usr/local/Cellar/composer/1.6.4/bin/composer/src/Composer/Console/Application.php:251
 Composer\Console\Application->doRun() at phar:///usr/local/Cellar/composer/1.6.4/bin/composer/vendor/symfony/console/Application.php:117
 Symfony\Component\Console\Application->run() at phar:///usr/local/Cellar/composer/1.6.4/bin/composer/src/Composer/Console/Application.php:100
 Composer\Console\Application->run() at phar:///usr/local/Cellar/composer/1.6.4/bin/composer/bin/composer:59
 require() at /usr/local/Cellar/composer/1.6.4/bin/composer:24
```

**Comment by Webonaute** [comment]
by the way, the repo is https://github.com/jmespath/jmespath.php.git

**Comment by nicolas-grekas** [comment]
Are you sure you ran the latest version of the package? I feel like I already fixed that issue...

**Comment by nicolas-grekas** [comment]
Thanks for the report, fixed by #60.
